.github/workflows/pathogen-repo-build.yaml.in

```diff
@@ -218,6 +218,16 @@ jobs:
           | "$NEXTSTRAIN_GITHUB_DIR"/bin/json-to-envvars
           | tee -a "$GITHUB_ENV"
 
+      - name: Create runtime AWS IAM inline session policy
+        env:
+          FULL_REPO_NAME: ${{ inputs.repo }}
+        run: |
+          export REPO_NAME="${FULL_REPO_NAME#*/}"
+          export PREFIX="${REPO_NAME}_"
```

Comment:
A prefix of `${name}_` is not suitable for all repos: it's suitable for repos which don't publish a top-level dataset (e.g. seasonal-flu doesn't publish a `seasonal-flu.json`), but some repos do publish that (e.g. zika publishes `zika.json`).

In my prototype, I'd [maintained prefixes explicitly for each repo](https://github.com/nextstrain/infra/commit/b1ceb76c795468142a00a68e4bc0c3b46f394839#diff-b528fba38d016216593468f87592430caee33f02da4ae20e909a09d8ee1e2937R1-R21) in order to account for per-repo variation here.

Reply:
Ah, that's true. So, this should probably just be another input.

Reply:
Alternatively, to avoid another input (yes, please!), the allowed patterns could always be both:

```
${PREFIX}.json
${PREFIX}_*.json
```

Yeah?

Reply:
Yeah, always allowing both makes sense to me.

Reply:
I'll be allowing both over on the AWS side.